x-pack/plugins/security_solution/public/flyout/translations.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ERROR_TITLE = (title: string) =>
```

Comment:
@PhilippeOberti i can foresee many components will have the same error message format, i made it flexible here, what do you think?